nanoFramework.System.Text.Benchmark/StringBenchmark.cs

```diff
@@ -0,0 +1,1048 @@
+
+using nanoFramework.Benchmark;
+using nanoFramework.Benchmark.Attributes;
+using System;
+using System.Collections;
+using System.Diagnostics;
+using System.Text;
+
+namespace nanoFramework.System.Text.Benchmark
+{
+    [IterationCount(100)]
```

Comment:
you will have to adjust the iteration and few more things but ll up, good direction!

Reply:
How would you like it adjusted?

Reply:
Should be good in fact, mixed up the base and the iteration. Should be fine.